File: app/model/UserResponseCredentials.ts
export class UserResponseCredentials {
  connectedUser:number;
  id: number;
  response: string;


  constructor( connectedUser:number,id: number, response: string) {
    this.connectedUser=connectedUser;
    this.id=id;
    this.response=response;

  }
}

File: src/main/java/com/learning_platform/learning_resources/dto/UserResponseCredentials.java
package com.learning_platform.learning_resources.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UserResponseCredentials {
    private Long connectedUser;
    private Long id;
    private String response;
}
